M2/Macaulay2/packages/Macaulay2Doc/functions/sign-doc.m2

```diff
@@ -0,0 +1,26 @@
+doc ///
+  Key
+    sign
```

Comment:
If this is a shared symbol, then it should be documented similar to other shared symbols, with links to nodes in other packages that enhance it.